comms/server/server.go

```diff
@@ -281,7 +281,10 @@ func createServer() *echo.Echo {
 		}
 
 		// call validator
-		rpcz.Validate(userId, rawRpc)
+		err = rpcz.Validate(userId, rawRpc)
```

Comment:
We validate again in [rpcz.Apply()](https://github.com/AudiusProject/audius-protocol/blob/fef95fc58beaddeb692243225c8179ae049936a7/comms/internal/rpcz/apply.go#L52)... should we remove that and keep only this invocation?

Reply:
I think Apply should call validate again.   When processing messages we should re-validate all the stuff: like signature + schema.